src/styles/atom-one-dark-reasonable.css

```diff
@@ -0,0 +1,77 @@
+/*
+
+Atom One Dark With support for ReasonML by Gidi Morris, based off work by Daniel Gamage
+
+Original One Dark Syntax theme from https://github.com/atom/one-dark-syntax
```

Comment:
It's unclear if you've made a derivative work from the Atom One theme here. If yes, please make sure you include the required copyright statement from Atom here. Please see: 
https://github.com/atom/one-dark-syntax/blob/master/LICENSE.md

Reply:
Hi Marco,
Thanks for taking the time to look at the PR. :)

Sorry if I wasn't clear - this is a derivative of the atom-one-dark theme that is already in the styles folder.
I assumed that as a derivative of a file already in the repo, there would be no need to add a new license.

Was I wrong?

Reply:
No, as long as you derived it from the files that were already in the project, then it should be fine.

Reply:
Brilliant, thanks